File: GazeChim.Data/Models/BsffFicheIntervention.cs
namespace GazeChim.Data.Models
{
    public class BsffFicheIntervention
    {
        public string Id { get; set; }
        public string Numero { get; set; }
        public float Weight { get; set; }
        public BsffDetenteur Detenteur { get; set; }
        public BsffOperateur Operateur { get; set; }
        public string PostalCode { get; set; }


    }
}

File: GazeChim/ClientApp/src/app/models/bsff/bsff-fiche-intervention.ts
import { BsffDetenteur } from './bsff-detenteur';
import { BsffOperateur } from './bsff-operateur';

export interface BsffFicheIntervention {
  id: string;
  numero: string;
  weight: number;
  detenteur: BsffDetenteur;
  operateur: BsffOperateur;
  postalCode: string;
}
